wrapper/src/main/java/com/moodysalem/phantomjs/wrapper/PhantomJS.java

```diff
@@ -233,28 +256,44 @@ public static PhantomJSExecutionResponse exec(InputStream script, PhantomJSOptio
     }
 
     /**
-     * LoggerOutputStream private nested class override
+     * Abstract logging class.
      */
-    private static class LoggerOutputStream extends LogOutputStream {
-        private Logger logger;
-        private Level level;
-        private StringBuffer messageContents;
-
-        LoggerOutputStream(Logger logger, Level level) {
-            super();
-            this.logger = logger;
-            this.level = level;
-            this.messageContents = new StringBuffer();
+    private static abstract class LoggerOutputStream extends LogOutputStream {
+
+        protected final StringBuffer messageContents;
+
+
+        LoggerOutputStream() {
+            messageContents = new StringBuffer();
         }
 
+
+        String getMessageContents() {
+            return messageContents.toString();
+        }
+    }
+
+    /**
+     * Info level logger.
+     */
+    private static class InfoLoggerOutputStream extends LoggerOutputStream {
+
         @Override
-        protected void processLine(String s, int i) {
-            logger.log(level, String.format("PhantomJS script logged: %s", s));
+        protected void processLine(final String s, final int i) {
+            logger.info("PhantomJS script logged: {}", s);
```

Comment:
The curly braces are the wildcard for what is %s in String.format()
Did you mean that or did I miss some variable?